src/paperless/settings.py

```diff
@@ -933,6 +935,19 @@ def default_threads_per_worker(task_workers) -> int:
 if TIKA_ENABLED:
     INSTALLED_APPS.append("paperless_tika.apps.PaperlessTikaConfig")
 
+AUDIT_LOG_ENABLED = __get_boolean("PAPERLESS_AUDIT_LOG_ENABLED", "NO")
+if AUDIT_LOG_ENABLED:
+    INSTALLED_APPS.append("auditlog")
+    MIDDLEWARE.append("auditlog.middleware.AuditlogMiddleware")
+db_conn = connections["default"]
+
+all_tables = db_conn.introspection.table_names()
+
+if ("auditlog_logentry" in all_tables) and not (AUDIT_LOG_ENABLED):
```

Comment:
This feels like it would work well as a system check instead.  This would also allow some basic testing to cover the lines